src/components/search_bar/query/ast_to_es_query_dsl.ts

```diff
@@ -43,28 +104,32 @@ export const _termValuesToQuery = (values, options) => {
   };
 };
 
-export const _fieldValuesToQuery = (field, operations, andOr) => {
-  const queries = [];
+export const _fieldValuesToQuery = (
+  field: string,
+  operations: { [x in OperatorType]: Value[] },
+  andOr: 'and' | 'or'
+) => {
+  const queries: QueryContainer[] = [];
 
-  Object.keys(operations).forEach(operator => {
+  (Object.keys(operations) as OperatorType[]).forEach(operator => {
```

Comment:
```suggestion
  keysOf(operations).forEach(operator => {
```

`keysOf` from _common.ts_

Reply:
I have no idea why I didn't do that to begin with.